doc/userguide/partials/commands-sc.rst

```diff
@@ -0,0 +1,112 @@
+.. Start with the most common basic commands.
+
+.. option:: shutdown
+
+   Shut Suricata instance down.
+
+.. option:: command-list
+
+   List available commands.
+
+.. option:: help
+
+   Get help about the available commands.
+
+.. option:: version
+
+   Print the version of Suricata instance.
+
+.. option:: uptime
+
+   Display the uptime of Suricata.
+
+.. option:: running-mode
+
+   Display running mode. This can either be *workers*, *autofp* or *simple*.
+
+.. option:: capture-mode
+
+   Display the capture mode. This can either be *UNIX_SOCKET* or
+   *AF_PACKET_DEV*.
```

Comment:
I think we can have more than that. Should be any of the '--list-runmodes' output.